### ISSUE TITLE
How to implement

### ISSUE THREAD

**Issue by Vinze** [shrimpwagon/laravel-route-controller#1]
I have followed the installation steps in the readme but it doesn't work for me in Laravel 5.4.

This is my routes/web.php file:

```
<?php
/*
|--------------------------------------------------------------------------
| Web Routes
|--------------------------------------------------------------------------
|
| Here is where you can register web routes for your application. These
| routes are loaded by the RouteServiceProvider within a group which
| contains the "web" middleware group. Now create something great!
|
*/

Route::controller('projects', 'ProjectController');
```

Now when I go to /projects I get the message: 

```
(1/1) InvalidArgumentException
Attribute [controller] does not exist.
```

Am I missing something?

**Comment by shrimpwagon** [comment]
Yes, I am having the same issue. Ive been trying to fix for the past few days. They made some changes in another class which was supporting this feature. I haven't quite figured it out. Ill get it working asap.

**Comment by Vinze** [comment]
Alright, thank you for the quick response! Hope you're able to fix this, good luck!

**Comment by jakus1** [comment]
I'd like to help fix this if you'd let me. Can you give me a rundown of what you have found?

**Comment by shrimpwagon** [comment]
@jakus1 You more very much more than welcome to fix.

I've been short on time to get this thing working 100%.

I've gotten it to where it seems to be adding the routes but when it goes to look for it in the route collection it does not find it.

I know this info probably doesn't help much.

I just need to jump back on this.

**Comment by jakus1** [comment]
@shrimpwagon, Can you send me your progress?

**Comment by shrimpwagon** [comment]
@jakus1 Yah, I can. I can't off top of my head though. I promise I'll dive back in today. I'll shoot you whatever I can come up with.

I actually need to do a new vanilla Laravel install and just go from there.

**Comment by jakus1** [comment]
Okay, thanks!

**Comment by shrimpwagon** [comment]
Oh... looks like someone already did this:

[laravel-advanced-route](https://github.com/lesichkovm/laravel-advanced-route)

Does that work?

**Comment by jakus1** [comment]
I'll try it. Thanks.

**Comment by shrimpwagon** [comment]
k, let me know. If that works I'm going to just delete this rep altogether.

**Comment by jakus1** [comment]
It almost works. It presupposes that all of your controllers exist in [namespace].'Http/Controllers/'
I may fork and fix it though. Are you needing it too?

**Comment by shrimpwagon** [comment]
I do need.

That would be awesome!

Yah, just fork that bad boy and we will go to town!

**Comment by jakus1** [comment]
With a small change that package will manage the controller routes, but you need to use the full namespaced controller. IE
```
AdvancedRoute::controller('shrimpwagon','App\Http\Controllers\ShrimpwagonsController');
```
